nixos/modules/services/hardware/monado.nix

```diff
@@ -0,0 +1,71 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+let
+  inherit (lib.modules) mkIf;
+  inherit (lib.options) mkEnableOption mkPackageOption;
+
+  cfg = config.services.monado;
+in
+{
+  options.services.monado = {
+    enable = mkEnableOption "monado";
```

Comment:
How exactly does this service work? I think it'd be a good idea to give the user a hint of what to do after enabling this.

Do you need to trigger monado to run with a command? By starting its service manually? Or does it start at login automatically?

I feel like that sort of thing should be explained in the description here. I'd just `// { description = ...; }` it.

Reply:
Monado is intended to be socket activated. So enabling this will only enable the monado socket which will start monado once a client connects to it.